src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -143,6 +147,18 @@ public override void ConfigureServices(IServiceCollection services)
         // Card Payment
         services.AddScoped<ICardPaymentService, CardPaymentService>();
         services.AddScoped<IDataMigration, StripeMigrations>();
+
+        // Exposing models to liquid tempaltes
+        services.Configure<TemplateOptions>(option =>
+        {
+            option.MemberAccessStrategy.Register<ShoppingCartViewModel>();
+            option.MemberAccessStrategy.Register<ShoppingCartCellViewModel>();
+            option.MemberAccessStrategy.Register<ShoppingCartLineViewModel>();
+            option.MemberAccessStrategy.Register<Amount, string>((obj, _) => obj.ToString());
+            option.MemberAccessStrategy.Register<Amount, decimal>((obj, _) => obj.Value);
+        })
+        // Liquid filter to convert JToken value to Amount struct in liquid.
+        .AddLiquidFilter<ToAmountConverterFilter>("toAmount");
```

Comment:
Liquid filters are supposed to be snake_case ([for example](https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/#absolute_url)). I think for conversion filters you are also just supposed to name it by the type, so `amount`.
Also please indent this.